### ISSUE TITLE
Controller support - can't select infiltration options

### ISSUE THREAD

**Issue by dadjumper** [long-war-2/lwotc#961]
Overall, controller works as it does in base WOTC, which I'm pleased with. 
One thing I've noticed you can't do is select the three options that come up once an infiltration is complete - "Boost infiltration," "View Squad" or "Abort."

They can be highligted with a controller, but as the A button is set to "Launch Mission," trying to select one of these options just launches the mission instead.

**Comment by kdm2k6** [comment]
I've noticed that, it appears, someone is working on controller implementation (I see "bind controller R3 to PlaceDelayedEvacZone" in the commit history) other than myself.

Therefore, just letting everyone know I'm working on this particular issue and am nearing completion.

**Comment by dadjumper** [comment]
Good to hear! I just noticed that the controller also can't select regions in the resistance management screen. Didn't want to open another issue so

**Comment by pledbrook** [comment]
I think that's covered by [this change](https://github.com/long-war-2/lwotc/pull/963) if I understand you correctly.

**Comment by pledbrook** [comment]
This was fixed by [commit 
53475b9](https://github.com/long-war-2/lwotc/commit/53475b95faf216fd52566bd754bf0f7c9c0a83d3)

**Comment by destroyer11687** [comment]
I understand this issue has been closed but I downloaded the commit and placed the adjusted files in my lwotc folder, but I’m still having this issue. @pledbrook is there something I’m not doing right to get this to work?
